Eatery/Controllers/EateryTabBarController.swift

```diff
@@ -54,7 +54,11 @@ extension EateryTabBarController: UITabBarControllerDelegate {
     func tabBarController(_ tabBarController: UITabBarController, shouldSelect viewController: UIViewController) -> Bool {
         if selectedViewController === viewController,
             let shared = viewController as? EateriesSharedViewController {
-            shared.activeViewController.scrollToTop(animated: true)
+            if shared.activeNavigationController.viewControllers.count > 1 {
```

Comment:
just wanna make sure I'm understanding what's happening here -- will admit I'm not very familiar with the eatery codebase/architecture. 

if there's currently an eatery detail view controller showing, and the user taps on the leftmost tab (the one for eateries), it'll pop back to the list of eateries, maintaining the scroll position that the user was in before selecting that eatery? otherwise, the list of eateries will scroll to the top of the list?

Reply:
That's the intended behavior. The music app does it this way for example